src/sass/nft-collection.scss

```diff
@@ -28,11 +75,71 @@
   &__right-side {
     width: 20%;
 
+    .nft-info {
```

Comment:
Move `.nft-info .nft-info-title .nft-info-header .nft-info-content .nft-info-footer and .nft-info-button` to the same level as `&__right-side`